.github/workflows/vast.yaml

```diff
@@ -65,6 +65,9 @@ jobs:
           apt-get -qq update
           apt-get -qqy install git libsnappy-dev libbrotli-dev zlib1g-dev wget liblz4-dev libzstd-dev ninja-build libpcap-dev libssl-dev libbenchmark-dev tcpdump lsb-release libflatbuffers-dev flatbuffers-compiler-dev libyaml-cpp-dev libsimdjson-dev gnupg2 gcc-8 g++-8 python3 python3-pip python3-venv jq apt-transport-https ca-certificates curl gnupg-agent software-properties-common
 
+          # spdlog
+          apt-get -qqy -t buster-backports install libspdlog-dev
```

Comment:
No need to do this with `-t buster-backports` now, the Dockerfile is running on Buster-backports now generally.

Reply:
The problem is that `libspdlog` has packages in both backports and the regular buster, and by default the non-backport one is selected:

```
root@5ab8414bcaf8:/# apt-cache policy libspdlog-dev
libspdlog-dev:
  Installed: (none)
  Candidate: 1:1.3.1-1
  Version table:
     1:1.5.0+ds-4~bpo10+1 100
        100 http://deb.debian.org/debian buster-backports/main amd64 Packages
     1:1.3.1-1 500
        500 http://deb.debian.org/debian buster/main amd64 Packages
root@5ab8414bcaf8:/# apt-get install libspdlog-dev
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following NEW packages will be installed:
  libspdlog-dev
0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
Need to get 176 kB of archives.
After this operation, 651 kB of additional disk space will be used.
Get:1 http://deb.debian.org/debian buster/main amd64 libspdlog-dev amd64 1:1.3.1-1 [176 kB]
[...]
```

Reply:
Can you add a comment about this to the Dockerfile for the future maintainer?